pkg/project/resource_project_repo_test.go

```diff
@@ -114,3 +116,128 @@ func TestAccProjectRepo(t *testing.T) {
 		},
 	})
 }
+
+/*
+Test to assign large number of repositories to a project
+*/
+func TestAccAssignMultipleReposInProject(t *testing.T) {
+
+	const numRepos = 100
+	const repoNameInitial = "repo-"
+
+	name := "tftestprojects" + randSeq(10)
+	resourceName := "project." + name
+	projectKey := strings.ToLower(randSeq(6))
+
+	repoNamesStr := func(repoCount int) string {
+		var repoNames []string
+		for i := 0; i < repoCount; i++ {
+			repoNames = append(repoNames, repoNameInitial+strconv.Itoa(i))
+		}
+		j, err := json.Marshal(repoNames)
+		if err != nil {
+			return "[]"
+		} else {
```

Comment:
We've discussed this before:

```go
if thing {
   return blah
}
return other
```
When you return from an `if` everything after the if **IS** the else

Reply:
cyclomatic complexity reduced